src/injector.ts

```diff
@@ -271,6 +299,9 @@ export default class Injector {
     files: string[]
   ) : Promise<void> {
 
+    this.validateAddresses(addresses);
```

Comment:
What if the address was left out on mainnet?

Reply:
We don't have an exception for mainnet in the current version. What made that possible in the first PoC?

Reply:
Ah sorry - that's what the addressDB is about. Going to open a separate issue about wiring that in.